nim_status/accounts/generator/generator.nim

```diff
@@ -64,9 +62,9 @@ proc deriveChildAccount(self: Generator, a: Account,
 proc deriveChildAccounts(self: Generator, a: Account,
   paths: seq[KeyPath]): DeriveChildAccountsResult =
 
-  var derived: seq[Account] = @[]
+  var derived = initTable[KeyPath, Account]()
   for path in paths:
-    derived.add ?self.deriveChildAccount(a, path)
+    derived[path] = ?self.deriveChildAccount(a, path)
 
   ok(derived)
```

Comment:
Super-minor comment: not related to this PR, but sometimes we use unqualified `Result.ok`, and sometimes qualified.

Reply:
Very relevant. This was a leftover from previous code. I will update it.

Reply:
Fixed.